components/BillHistoryTable/BillHistoryTable.js

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import { Table } from 'react-bootstrap'
+
+const DocumentHistoryActionRows = (documentHistoryActions) => {
```

Comment:
Need to wrap the documentHistoryActions in curly braces to extract it from the props object:

```
const DocumentHistoryActionRows = ({documentHistoryActions}) => {
```